### PR TITLE
Add markers and info popups to map

### PR DESCRIPTION
**Pull request by mollygraton**
Adding Markers to map of my hometown and popup windows that give context to the locations.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js
Comment (cisxingg) [inline portfolio/src/main/webapp/script.js]: This is fine as we don't have a lot of data here, another way to do this is:
1. make a constant map with coordinates and info.
2. Write a for loop to access the map and call addMarkerInfo in the for loop.

**Review by cisxingg — APPROVED** [review]
LG in general, just have one suggestion :)
Please wait for Warfa's review before merging.

**Review by warfa-google — APPROVED** [review]
LGTM

**Pull request merged**
